src/styles/main.scss

```diff
@@ -0,0 +1,35 @@
+.main {
+  justify-content: center;
+  margin: 0;
+  padding: 50px 40px;
+  display: grid;
+
+  --column-count: 4;
+
+  grid-template-columns: repeat(var(--column-count), 200px);
+  gap: 46px 48px;
```

Comment:
The gap between grid items is set to '46px 48px', which matches the task requirement for horizontal and vertical gaps.